additional_recipes/ros-humble-rmw-wasm-cpp/recipe.yaml

```diff
@@ -0,0 +1,96 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+package:
+  name: ros-humble-rmw-wasm-cpp
+  version: 1.1.0
```

Comment:
This is different from https://github.com/Tobias-Fischer/rmw_wasm/blob/emscripten-async-fixes/rmw_wasm_cpp/package.xml#L5 . If this is intended, feel free to ignore this message.

Reply:
It wasn't - fixed, thanks!